tools/manage_clvm.py

```diff
@@ -186,7 +202,8 @@ def main() -> None:
 
 @main.command()
 @click.option("--use-cache/--no-cache", default=True, show_default=True, envvar="USE_CACHE")
-def check(use_cache: bool) -> int:
+@click.option("--fix-hashfile-trailing-whitespace", default=True, show_default=True)
```

Comment:
To avoid confusion about why it exists and when to use it.  Why is the check now writing a file?  I guess you now want it fixing things whereas before it was specifically wanted to _not_ be writing the files at all?

Reply:
I added this option precisely because I understood the check to not want to edit any files.  I wasn't sure where the use case would be where you would not want it to edit files, but to preserve that use case I added this option.  If the whitespace isn't correct, the pre-commit check will fail because it will edit the file.

Reply:
I'm not clear why the check should fix only one thing.  The previous intent was specifically to have the check not fix stuff, just report issues.  Any issues were expected to be dealt with explicitly and directly by separate action by the developer.  This was part of the protection provided by never automatically modifying the files.  So, I would have expected the check to continue not editing files and a build, or manual edit, would be required to fix the files.

Reply:
Okay removed